packages/neo-one-node-http-rpc/src/middleware/tooBusyCheck.ts

```diff
@@ -8,15 +8,19 @@ export interface Options {
   readonly smoothingFactor?: number;
 }
 
-export const tooBusyCheck = (options: Options) => {
-  if (options.interval !== undefined) {
-    tooBusy.interval(options.interval);
-  }
-  if (options.maxLag !== undefined) {
-    tooBusy.maxLag(options.maxLag);
-  }
-  if (options.smoothingFactor !== undefined) {
-    tooBusy.smoothingFactor(options.smoothingFactor);
+tooBusy.maxLag(5000);
```

Comment:
With much testing, this was found to be the sweet spot for tests to run successfully.

Reply:
What's the default here? Which tests are causing this to trip?